CHANGELOG.md

```diff
@@ -0,0 +1,99 @@
+# v3.0.0
+
+## Key Changes
+
+The new `solana-exporter` (renamed from `solana_exporter`) contains many new metrics, standardised naming conventions 
+and more configurability.
+
+## What's Changed
```

Comment:
Might be worth explicitly calling out some of the new features under this section without a subsection

* fee reward tracking
* easy healthy / num_slots_behind stats
* tracking of active stake per specified validator
* balance monitoring for any arbitrary address/es
* block size tracking to monitor for blocks without user transactions from bad host or relayer configuration